### DISCUSSION TITLE
Release schedules

### DISCUSSION THREAD

**Discussion by johanvos** [gluonhq/scenebuilder#651] [General]
Hi,

In order to 
* make releases more predictable
* avoid PR's being moved to a next release because we're not 100% sure yet
* making sure enough testing is done before a release is going out of the door, 
 
I propose the following:

* We create a milestone for release X, which ends 6 weeks before the planned release of JavaFX X
* After that milestone, we create a release branch sb-X which is only updated when critical issues are observed with the milestone. This should be avoided as much as possible
* PR's can still be reviewed, and merged into the main branch (affecting release X + 1)
* We release Scene Builder X "shortly" after JavaFX X is released ("shortly" being defined as between 1 day and 2 weeks)

This ensures there is enough time to test merged PR's, and it avoids that we keep moving PR's to the next release because we don't dare to merge them shortly before a release.
This is more or less similar to what is done in OpenJFX, and it seems to work pretty well.

Thoughts?

**Comment by Oliver-Loeffler** [comment]
Hi, I appreciate and like this approach, this will make things more predictable and plannable.
